coffee/rewriter.py

```diff
@@ -143,7 +143,7 @@ def expand(self, mode='standard'):
                      * mode == 'full': expansion is performed aggressively without \
                                        any specific restrictions.
         """
-        symbols = FindInstances(Symbol).visit(self.stmt.children[1])[Symbol]
+        symbols = FindInstances(Symbol).visit(self.stmt.children[1], ret=FindInstances.default_retval())[Symbol]
```

Comment:
I always try to have lines shorter than 90 cols (*rarely* I get to ~93-94), so we should stick to it